src/services/ArticleService.ts

```diff
@@ -0,0 +1,46 @@
+import axios from "axios";
+import ArticlePreview from "../interfaces/ArticlePreview";
+
+class ArticleService {
+	private static instance: ArticleService;
+
+	/* eslint-disable */
+	private constructor() { }
+	/* eslint-enable */
+
+	static getInstance(): ArticleService {
+		if (!this.instance) {
+			this.instance = new ArticleService();
+		}
+
+		return this.instance;
+	}
+
+	async getLatest(amount: number): Promise<ArticlePreview[]> {
```

Comment:
Could this be `getLatestArticles` please?

Reply:
Mhm, GithubService also doesn't do `getGithubIssues`. It does `getIssues`.... why should this have the name of the service in the method?

Reply:
Fair enough, you can keep it as is then.